### PR TITLE
Bootstrap project Milestone 2

### PR DESCRIPTION
**Pull request by slavkojos**
Here is the code for Milestone 1. We started using bootstrap framework and so far we implemented header and footer.
Any comments are appreciated.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+.main-header-right-align {
+    font-family: Arial, Helvetica, sans-serif;
+    font-weight: 700;
+}
+
+.login {
+    text-decoration: none;
+    color: white;
+}
```
Comment (ghost) [inline styles.css]: Avoid using custom styling where bootstrap styling is possible. Example in the above, you can use `text-decoration-none` and `text-white` bootstrap classes. Please check the rest of your styling as well.

**Review by ghost — APPROVED** [review]
# Status: Approved 🎉

## Hello team 👋,

**Great job on this milestone 👏.**

## Highlights.

- **You have installed Bootstrap properly.**
- **Your webpage is responsive.**

## Improvements to note moving forward

- **Fix the linter error checks.**
- **Ensure the spacing and positioning of elements match that of the original page for all screen sizes. Example in the Navbar section, the Navbar logo should be centred for desktop screen size and aligned to the left for medium and small screen-sizes.**
- **Use a `nav` tag for your navigation menu.**
- **The Newsweek logo and social media icons, at the footer section, should be links.**
- **Check one more review in the comment below.**

## Happy coding 🥂.

**You can reach me on slack (Rose Wanjohi).**

**Comment by slavkojos** [comment]
added main content still needs some styling

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+.main-header-right-align {
+  font-family: Arial, Helvetica, sans-serif;
+  font-weight: 700;
+}
+
+.login {
+  text-decoration: none;
+  color: white;
+}
```
Comment (ghost) [inline styles.css]: Avoid using custom styling where bootstrap styling is possible. Example in the above, you can use `text-decoration-none` and `text-white` bootstrap classes. Please check the rest of your styling as well.

**Review by ghost — CHANGES_REQUESTED** [review]
# Status: Changes Required 🛑

## Hello team 👋

**Great job on this milestone 👏.**

## Highlights

- **Your linter checks are working properly.**

## Improvements to note moving forward ▶️ 

- **Add a getting started section in your READme file with details on how to set up your project.**
- **Column one, for desktop screen size, is missing the Sponsored insights / Rankings section.**
- **Column two, for desktop screen size, is missing the Latest news section.**
- **Images in Featured Stories, Top Story, Cultural & Travel and Sponsored sections should have labels at the bottom leftmost edge of every image.**
- **Improve the design of your page so it looks like the original. Example, Most titles should be bold and every content in the more stories section has a top and bottom border.**
- **Fix the order in which your sections display in the mobile screen size so it matches that of the original page. For example, the section at the topmost page should be Top Story followed by Featured Stories, The Debate... You started with the Featured Stories, then Top Story, My Turn...**
- **All images and titles should be links.**
- **Check one more in the comment below.**

## Happy coding 🥂.

**You can reach me on slack (Rose Wanjohi).**

**Review by PhillipUg — CHANGES_REQUESTED** [review]
Hey team, 👋

You have done an amazing job implementing the requirements for this project.

However, there are still some changes 👇 you need to implement to approve this request 🔍

### Changes Required ♻️ 

- [x] The Newsweek logo in the footer and the social media icons should have links
- [x] On tablet screen size, the section above the `In The Magazine` section, should be centered.
- [x]  Your titles throughout the page should use hx tags (h1 - h6). I notice that you're using Divs and anchor tags.

#### Please, make all REQUIRED changes and submit your request again! 😀

If you have any questions, feel free to send me a message through Slack as **Phillip Musiime** or
 <a href="mailto:phillipmusiime@gmail.com"> gmail</a> &nbsp; &nbsp; [Twitter Follow](https://twitter.com/intent/follow?screen_name=Phillip_Ug) &nbsp; &nbsp; [GitHub followers](https://github.com/PhillipUg)

**Review by SpaYco — APPROVED** [review]
# Project Approved ✔️ 

thank you for making the required changes, your project is now approved!

### please don't forget to merge to master, good luck with the next projects!


Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Pull request merged**